lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -185,6 +185,14 @@
             - When a default security group is created for a Linux host a rule will be added allowing inbound TCP
               connections to the default SSH port 22, and for a Windows host rules will be added allowing inbound
               access to RDP ports 3389 and 5986. Override the default ports by providing a list of open ports.
+    enable_ip_forwarding:
+        description:
+            - Whether to enable IP forwarding
+        aliases:
+            - ip_forwarding
```

Comment:
Why adding an alias?

Reply:
Why not? It seems like a reasonable alias. Can be removed if desired.